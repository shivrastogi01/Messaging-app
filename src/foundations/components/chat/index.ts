export * from "./text-message/text-message";
export * from "./image-message/image-message";
export * from "./link-message/link-message";
export * from "./giphy-message/giphy-message";
export * from "./avatar/avatar";
