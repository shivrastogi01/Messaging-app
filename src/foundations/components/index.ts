export * from "./presentation";
export * from "./layout";
