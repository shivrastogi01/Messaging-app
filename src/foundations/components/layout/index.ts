export * from "./styled-box/styled-box";
export * from "./scroll-view/scroll-view";
export * from "./flex-row/flex-row";
export * from "./flex-column/flex-column";
export * from "./modal/modal";
export * from "./drawer/drawer";
export * from "./list-item/list-item";
export * from "./dropdown/dropdown";
