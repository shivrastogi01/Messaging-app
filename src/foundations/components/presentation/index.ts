export * from "./heading/heading";
export * from "./label/label";
export * from "./title/title";
export * from "./icon/icon";
export * from "./button/button";
export * from "./input/input";
export * from "./textarea/textarea";
