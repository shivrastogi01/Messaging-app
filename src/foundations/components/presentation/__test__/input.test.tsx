import React from "react";
import renderWithTheme from "main/testing/render-with-theme";
import { Input } from "foundations/components/presentation";

describe("Input", () => {
  test("default matches the snapshot", () => {
    const { container } = renderWithTheme(<Input />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
