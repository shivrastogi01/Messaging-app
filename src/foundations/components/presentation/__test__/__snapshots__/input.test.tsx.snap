// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input default matches the snapshot 1`] = `
.c0 {
  border: 1px solid #9B9B9B;
  border-radius: 5px;
  color: #585858;
  height: 42px;
  padding: 14px;
  width: 100%;
}

.c0[type="password"] {
  font-size: 24px;
  -webkit-letter-spacing: 8px;
  -moz-letter-spacing: 8px;
  -ms-letter-spacing: 8px;
  letter-spacing: 8px;
}

.c0:focus {
  outline: none;
}

<input
  class="c0"
/>
`;
